python/run.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+
+class Person(object):
+    def __init__(self):
+        pass
+
+    def has_a_beard(self):
+        return self.facial_hair_length > 2 and self.on_chin
```

Comment:
why 2?

Reply:
yeah, i haven't actually written the method yet ---- which is interesting because all by one test passes now!